### ISSUE TITLE
Load specified loss function in config.yml.

### ISSUE THREAD

**Issue by yutayamazaki** [yutayamazaki/semantic-segmentation-pytorch#4]
- Add `load_loss` function and its tests to load specified loss.
- Use `load_loss` in `run_train.py`.
